index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
```

Comment:
Please specify a viewport meta element.

Reply:
> Please specify a viewport meta element.

- [ ] Kindly explain the why of your comments to make sure the person you are reviewing understand why they're going to make this changes :+1: